composer.json

```diff
@@ -7,13 +7,19 @@
     "autoload-dev": {
         "psr-4": {
             "App\\Spec\\": "spec/"
-        }
+        },
+        "files": [
+          "src/prodecural.php"
+        ]
     },
     "config": {
         "bin-dir": "bin"
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
+    "require": {
+      "php": ">=7.1"
```

Comment:
Kahlan still use at least php 5.4. And at least it still uses php 5.5 for this repo right now. For specific use case, you can do:

```php
skipIf(PHP_MAJOR_VERSION < 7);
```

or 

```
skipIf(PHP_MAJOR_VERSION < 5.6);
```
in tests.